cmsArticle.php

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<?php
+    require_once 'dropdownDB.php';
+    require_once 'getArticleImgDB.php';
+    require_once 'cmsUpdateArticleDB.php';
+?>
+
+
+<head>
+    <meta charset="UTF-8">
+    <title>CMS About Page</title>
+    <link href="style.css" rel="stylesheet" type="text/css">
+    <link href="normalize.css" rel="stylesheet" type="text/css">
+</head>
+
+<body>
+<h1>Edit Article Page - About Me and Portfolio</h1>
+
+    <form method="post" action="cmsArticle.php">
+        <select name="selectedTitle">
+            <?php
+            echo $aboutDropdown;
+            ?>
+        </select>
+        <input style="margin: 10px" type="submit" value="Pull from DB">
+    </form>
+
+    <div>
+        <?php $articleDB_id = getArticleFromDB()[0]['id'];?>
```

Comment:
if you're always only selecting `getArticleFromDB()[0]` you could use the `fetch()` method rather than `fetchAll()`